### PR TITLE
[INLONG-10370][Manager] Support configuration of kV data format

### LINKED ISSUE #10370: [Improve][Manager] Support configuration of kV data format
### Description

Support configuration of kV data format.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
Fixes #10370

### Motivation

Support configuration of kV data format.

### Modifications

Support configuration of kV data format.

\

### CODE REVIEW

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/node/base/ExtractNodeProvider.java [hunk 1]
```diff
@@ -131,6 +136,21 @@ default Format parsingFormat(
             case RAW:
                 format = new RawFormat();
                 break;
+            case KV:
+                if (StringUtils.isNumeric(separatorStr)) {
```
Comment (vernedeng) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/node/base/ExtractNodeProvider.java:140]: Extract to one method

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/kafka/KafkaSource.java [hunk 2]
```diff
@@ -90,6 +90,9 @@ public class KafkaSource extends StreamSource {
     @ApiModelProperty(value = "Data separator")
     private String dataSeparator;
 
+    @ApiModelProperty(value = "Kv separator")
```
Comment (vernedeng) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/kafka/KafkaSource.java:93]: KV

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/kafka/KafkaSourceRequest.java [hunk 3]
```diff
@@ -80,10 +78,13 @@ public class KafkaSourceRequest extends SourceRequest {
...
 
     @ApiModelProperty(value = "Data encoding format: UTF-8, GBK")
-    private String dataEncoding = StandardCharsets.UTF_8.toString();
+    private String dataEncoding;
 
     @ApiModelProperty(value = "Data separator")
-    private String dataSeparator = String.valueOf((int) '|');
+    private String dataSeparator;
+
+    @ApiModelProperty(value = "Kv separator")
```
Comment (vernedeng) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/kafka/KafkaSourceRequest.java:86]: ditto

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/pulsar/PulsarSource.java [hunk 4]
```diff
@@ -73,6 +73,9 @@ public class PulsarSource extends StreamSource {
     @ApiModelProperty(value = "Data separator")
     private String dataSeparator;
 
+    @ApiModelProperty(value = "Kv separator")
```
Comment (vernedeng) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/pulsar/PulsarSource.java:76]: ditto

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/node/base/ExtractNodeProvider.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/node/base/ExtractNodeProvider.java]: Fixed.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/pulsar/PulsarSource.java (hunk 4 above)
Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/pulsar/PulsarSource.java]: Fixed.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/kafka/KafkaSource.java (hunk 2 above)
Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/kafka/KafkaSource.java]: Fixed.

**Review by vernedeng — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Pull request merged**